capstone/src/main/webapp/login_script.js

```diff
@@ -6,31 +7,31 @@ function onSignIn(googleUser) {
 }
 
 function signOut() {
-    var auth2 = gapi.auth2.getAuthInstance();
-    auth2.signOut().then(function () {
-      console.log('User signed out.');
-    });
-    location.reload();
+  var auth2 = gapi.auth2.getAuthInstance();
+  auth2.signOut().then(function () {
+    console.log('User signed out.');
+  });
+  location.reload();
 }
 
 function isSignedIn() {
   return gapi.auth2.getAuthInstance().isSignedIn.get();
```

Comment:
Does this call do any caching if it is called multiple times? Or does it send an external request every time? If so, are there any secure approaches to avoid having to make an external call every time?